src/quilla/browser/browser_validations.py

```diff
@@ -116,6 +119,12 @@ def validate(self) -> List[BaseReport]:
         except Exception as e:
             # Catch exception and crash the program, but clean up
             # the browser first
+            self.ctx.logger.debug(
```

Comment:
Why it's debug level not error level?

Reply:
I originally thought it might make more sense leaving it as a debug so it wouldn't show up normally, but since the stack trace will be shown to the user regardless (since nothing catches the re-raised exception), you're probably right about it being better as an error-level